### PR TITLE
AEROGEAR-3533 move and rename keycloak setup

### PR DESCRIPTION
**Pull request by finp**

### CODE REVIEW

**Comment by finp** [comment]
ping @pawelpaszki

File: modules/ROOT/pages/keycloak/keycloak-setup.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //include::{partialsdir}/attributes.adoc[]
 
-= Keycloak Setup
+= Setting Up {keycloak-service}
```
Comment (laurafitzgerald) [inline modules/ROOT/pages/keycloak/keycloak-setup.adoc]: When I view this in the Identity Managment Service The heading is not at the same level as other headings. e.g The Binding a Mobile Client, I've included a screenshot 
image

Comment (laurafitzgerald) [reply modules/ROOT/pages/keycloak/keycloak-setup.adoc]: However it works fine in the getting-started guide 
image

**Comment by finp** [comment]
closing due to comments in [AEROGEAR-3524]. however @pawelpaszki  some of Laura's comments might be relevant there

**Comment by laurafitzgerald** [comment]
@finp looks like the JIRA you mentioned it links to this pr so not sure where I should put the comments that I've included in this pr. Is there a new pr somewhere?

**Comment by johnfriz** [comment]
@laurafitzgerald - Please review https://github.com/aerogear/mobile-docs/pull/157 and add any comments there.

**Pull request closed without merging**